src/pipeline/LevelAdjustStep.cpp

```diff
@@ -57,7 +58,9 @@ std::shared_ptr<short> LevelAdjustStep::execute(std::shared_ptr<short> inputSamp
 
     for (int index = 0; index < numInputSamples; index++)
     {
-        outputSamples_.get()[index] = inputSamples.get()[index] * scaleFactor;
+        double y = inputSamples.get()[index] * scaleFactor;
+        y = 32767.0 * std::tanh(y / 32767.0);  // Soft-Clip statt hartem Clipping
```

Comment:
I always get concerned when we start to add non-linearity, especially if is intended to prevent clipping a digital waveform. Can we disable this for the TX level slider but leave it for the mic/speaker volume one?